### DISCUSSION TITLE
Is there a release schedule?

### DISCUSSION THREAD

**Discussion by mcc138** [jitsi-contrib/jitsi-helm#46] [Q&A] [answered]
Hi,
is there a release schedule, if not when do you plan to build the next release?
I have made a couple of pull requests which are accepted into the main branch but are not released.
Currently I have to patch by hand with every deployment
Thanks
Matt

**Comment by sapkra** [comment, marked as answer]
There is no schedule. I'm just doing it from time to time. If someone needs a faster release they should mention me somehow. I just released v1.2.2.

If you want faster released I would appreciate a PR to create them automatically using GitHub Actions.

**Reply by mcc138** [reply]
Great thank you, I will create PRs in the future :)
